### PR TITLE
added release notes content for the  release 77908-xxxx

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

Added contents for release notes 77908

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/service-reliability-management.md [hunk 1]
```diff
@@ -13,6 +13,31 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+### Early Access Features
+
+This release does not include early access features.
+
+### Fixed Issues
+
+Service Health screen displays service status as Healthy even if there is no metric data available from the health source for a time period. (SRM–13336)
+
+Now, the Service Health screen displays the service status as NO_DATA if there is no metric data available from the health source for a time period.
```
Comment (rashminandasahoo) [inline release-notes/service-reliability-management.md:34]: Please see if this looks correct in the context "When there are no metrics available from the health source, the Service Health screen now shows the service state as `NO DATA`."

File: release-notes/service-reliability-management.md [hunk 2]
```diff
@@ -13,6 +13,31 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+Saving the **Verify** step in a Stage template throws the `Invalid YAML. Can't find pipeline.` error if the YAML file does not contain a pipeline key. (SRM-13320)
```
Comment (rashminandasahoo) [inline release-notes/service-reliability-management.md:36]: Saving the **Verify** step in a stage template throws an error if the pipeline key is missing in the YAML.

File: release-notes/service-reliability-management.md [hunk 3]
```diff
@@ -13,6 +13,31 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+ 
+You can now save the **Verfy** step in a Stage template even if the YAML file does not contain a pipeline key.
```
Comment (rashminandasahoo) [inline release-notes/service-reliability-management.md:38]: Typo in Verify.

**Review by rashminandasahoo — CHANGES_REQUESTED** [review]

File: release-notes/service-reliability-management.md (hunk 1 above)
Comment (SudheendraKatte) [reply release-notes/service-reliability-management.md:34]: Thanks for the suggestion. I prefer to keep the existing text as the suggested text makes the information technically not complete.

File: release-notes/service-reliability-management.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply release-notes/service-reliability-management.md:38]: The fix contains backend information which is not exposed to the end user.

File: release-notes/service-reliability-management.md [hunk 4]
```diff
@@ -13,6 +13,31 @@ Harness deploys updates progressively to different Harness SaaS clusters. You ca
...
+
+This release does not include enhancements.
+
+### Early Access Features
+
+This release does not include early access features.
+
+### Fixed Issues
+
+Service Health screen displays service status as Healthy even if there is no metric data available from the health source for a time period. (SRM–13336)
```
[the remark this answers is not in the document]
Comment (SudheendraKatte) [reply release-notes/service-reliability-management.md:32]: Here is the updated text:

- Even though metric data is not available from the health source for a time period, the Overall Health Score timeline displays green bars and shows the service score as 100.
  
  Now, on the Service Health page, the Overall Health Score timeline displays grey bars if metric data is unavailable from the health source for a specific time period. It does not show the health score.

**Review by rashminandasahoo — APPROVED** [review]

**Pull request merged**